cmd/storage-rest-client.go

```diff
@@ -856,10 +856,7 @@ func (client *storageRESTClient) CleanAbandonedData(ctx context.Context, volume
 		return err
 	}
 	defer xhttp.DrainBody(respBody)
-	respReader, err := waitForHTTPResponse(respBody)
-	if err == nil {
-		io.Copy(io.Discard, respReader)
```

Comment:
Not sure discarding the body was removed here in particular

Reply:
body discarding happens a line above.